packages/next/src/client/components/is-hydration-error.ts

```diff
@@ -3,7 +3,7 @@ import isError from '../../lib/is-error'
 const hydrationErrorRegex =
   /hydration failed|while hydrating|content does not match|did not match/i
 
-const reactUnifiedMismatchWarning = `Hydration failed because the server rendered HTML didn't match the client. As a result this tree will be regenerated on the client. This can happen if a SSR-ed Client Component used`
+const reactUnifiedMismatchWarning = `Hydration failed because the server rendered HTML didn't match the client. As a result this tree will be regenerated on the client. This can happen when an SSR-ed Client Component is used.`
```

Comment:
This sentence must be either kept as is or deleted (in case the React team ever decides to fix the grammar). Otherwise, detecting the [React error message](https://github.com/facebook/react/blob/e2e12f33517e528bc955fe3f0098d6765f4648af/packages/react-reconciler/src/ReactFiberHydrationContext.js#L319) in the stack would fail.